src/org/opendatakit/briefcase/util/FindDirectoryStructure.java

```diff
@@ -44,12 +44,18 @@ public static boolean isMac() {
     String os = System.getProperty(PROPERTY_OS);
     return os.contains(OS_MAC);
   }
-  
+
+  public static boolean isUnix() {
+    // Copied from https://www.mkyong.com/java/how-to-detect-os-in-java-systemgetpropertyosname/
```

Comment:
Maybe it was inspired by the mkyong site, but I would remove this comment because it's not a copy. 

As an aside, unless the code on the site you look at is explicitly licensed, we can't use it. That is public code != open source code. We have this policy detailed in https://github.com/opendatakit/briefcase/blob/master/CONTRIBUTING.md#code-from-external-sources.